.github/workflows/proto-registry.yml

```diff
@@ -0,0 +1,74 @@
+name: Third Party Proto Registry
+# Protobuf runs buf (https://buf.build/) push updated proto files to https://buf.build/provenance-io
+# This workflow pushes the third party protos to buf.build whenever any of them are updated in `main`.
+on:
+  push:
```

Comment:
Oh! Do we want to add `workflow_dispatch` in here too: [events-that-trigger-workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch)? The we have the ability to manually kick off the action if we ever need to.

I've never tried it though.

Reply:
Neither have I. The docs state that it needs to be merged to `main` (default branch) before it can be manually triggered but am not sure if it's restricted to only run on `main` after that. See step 5 in [manually-running-a-workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow). As long as we can only run it on main then we're fine.

I've enabled it but let me run a test on a test repo first before we merge it.